src/features/leaderboard/components/tablecontent/index.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { Table, Typography, Space, Tabs, Spin, Image } from 'antd';
+import { ScoresRepositoryContext } from 'context/scores';
+import { DancersRepositoryContext } from 'context/dancer';
+import { DefaultAllDancers } from 'context/dancer/constants';
+import { DefaultScore } from 'context/scores/constants';
+import { SongsRepositoryContext } from 'context/songs';
+import { DefaultSong } from 'context/songs/constants';
+import { LeaderboardContainer } from './styled';
+
+const { Title, Paragraph } = Typography;
+const { TabPane } = Tabs;
+
+const TableContent = () => {
+  const songsRepo = useContext(SongsRepositoryContext);
+  const scoresRepo = useContext(ScoresRepositoryContext);
+  const dancersRepo = useContext(DancersRepositoryContext);
+  const [scores, setScores] = useState(Array(DefaultScore));
+  const [songs, setSongs] = useState(Array(DefaultSong));
+  const [allDancers, setAllDancers] = useState(Array(DefaultAllDancers));
```

Comment:
These are probably better done in the form:
`const [thing, setThing] = useState<Array<thingType>>(null);`
You will be able to assert truth on it (`if (thing)`) this way.